pkg/crc/machine/start.go

```diff
@@ -242,10 +242,10 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 			return nil, errors.Wrap(err, "Failed to ask for pull secret")
 		}
 
-		if crcBundleMetadata.IsOpenShift() {
-			logging.Infof("Creating CRC VM for %s %s...", startConfig.Preset, crcBundleMetadata.GetOpenshiftVersion())
+		if crcBundleMetadata.IsPodman() {
+			logging.Infof("Creating CRC VM for %s %s...", startConfig.Preset, crcBundleMetadata.GetPodmanVersion())
 		} else {
-			logging.Infof("Creating CRC VM for Podman %s...", crcBundleMetadata.GetPodmanVersion())
+			logging.Infof("Creating CRC VM for %s %s...", startConfig.Preset, crcBundleMetadata.GetOpenshiftVersion())
```

Comment:
Untested for now, but something like this should work too:
https://github.com/cfergeau/crc/commit/fcd0de25f6e8bea6414798e73f127badcee84cd5

Reply:
tested and that works, cherry-picked this commit